tests/console/consoletest_setup.pm

```diff
@@ -21,6 +21,9 @@ sub run {
     # let's see how it looks at the beginning
     save_screenshot;
 
+    # set kernel loglevel to NOTICE to prevent interference
+    sendkey 'alt-sysrq-5';
```

Comment:
No way. Please take a look into tests/console/force_cron_run first how it's done there and please also read the story behind the bugs mentioned there. This is definitely part of a way bigger story. If you want more information after reading you can also talk to @rwx788, me and @coolo but we should definitely not hide these problems from the tests. The tests should not hide broken product behaviour.

Reply:
Once you do hit a bug, these debug messages are the only thing that help you.

Reply:
but I can buy the argument to disable it before first login - and then reenabling it.